PowerPointLabs/PowerPointLabs/AgendaLab/AgendaLabMain.cs

```diff
@@ -9,6 +9,7 @@
 using System.Windows.Forms;
 using Microsoft.Office.Core;
 using Microsoft.Office.Interop.PowerPoint;
+using PowerPointLabs.AgendaLab.Templates;
```

Comment:
Add a newline to group between .NET namespaces and other namespaces?
(Some files follow this, but many do not)
https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-CSharp.md#2-place-using-directives-at-the-top-of-the-file-grouped-and-ordered-alphabetically